app/upgrades_test.go

```diff
@@ -896,9 +896,9 @@ func (s *AppTestSuite) TestV010505UpgradeHandler() {
 func (s *AppTestSuite) TestV010601UpgradeHandler() {
 	s.InitV160TestZones()
 	app := s.GetQuicksilverApp(s.chainA)
-
 	ctx := s.chainA.GetContext()
-
+	// upgrade only run on production chain
+	ctx = ctx.WithChainID(upgrades.ProductionChainID)
```

Comment:
A better fix would be to make the IsMainnet() check in the upgrade to a IsMainnet() || IsTest()

Reply:
@joe-bowman I guess adjusting the test for local, testnet or mainnet a better pattern? In my opinion upgrade handler should not take the test into account.

Reply:
@joe-bowman but the way we setup the chain, the chainID would not be testnet1, thus isTest() will return false

Reply:
Hey @joe-bowman IsTest() won't work since the chain id here is not `testnet2` but some chain-id provided by the ibc test suite. I think my solution/workaround is good for now.